### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by IIILIGHTIII**
https://IIILIGHTIII.github.io/layout_moyo-header/
https://IIILIGHTIII.github.io/layout_moyo-header/report/html_report/SD

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,52 @@
...
+          <li class="list__item">
+            <a href="#" class="item">Smartphones</a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="item"
+              data-qa="hover"
+            >
+            Laptops & computers
```
Comment (mykhalenych) [inline src/index.html:36]:
```suggestion
              Laptops & computers
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,76 @@
...
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  line-height: 14px;
+  padding: 23px 0;
+  cursor: pointer;
+}
+
+.is-active {
+  color: #00acdc;
```
Comment (mykhalenych) [inline src/style.css:49]: when you use color more than 1 time better to create variable and use variable instead

**Review by mykhalenych — CHANGES_REQUESTED** [review]
your test link isn't working

**Review by IIILIGHTIII — COMMENTED** [review]
https://IIILIGHTIII.github.io/layout_moyo-header/
https://IIILIGHTIII.github.io/layout_moyo-header/report/html_report/

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**